src/Payments.Mvc/Controllers/CouponsController.cs

```diff
@@ -45,6 +45,12 @@ public async Task<IActionResult> Create(CreateCouponViewModel model)
                 Team            = team
             };
 
+            if (team.Coupons.Any(a => a.Code != null && a.Code.Equals(coupon.Code, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Should this check also exclude coupons that are expired?

Reply:
@sprucely I don't think so. Going forward, we will require unique coupon codes. I'm just trying to change this as little as possible now to avoid introducing bugs.